commit 5fe9855624c1a998eb57a1123a5b358646f3914b
Author: Slava Sysoltsev <SlavaSysoltsev@gmx.de>
Date:   Fri Mar 1 08:02:22 2013 -0500

    FindImageMagick: Search quantum depth suffixes (#13859)
    
    From ImageMagick's 6.8.0-8 changelog:
    
      http://www.imagemagick.org/script/changelog.php
      ABI is incompatible if quantum depth change.
      Add abi indication to library name.
    
    Search for library names with -Q16 and -Q8 suffixes.
    
    Reported-by: Evangelos Foutras <evangelos@foutrelis.com>

diff --git a/Modules/FindImageMagick.cmake b/Modules/FindImageMagick.cmake
index bf09616..4aaefb7 100644
--- a/Modules/FindImageMagick.cmake
+++ b/Modules/FindImageMagick.cmake
@@ -147,17 +147,17 @@ foreach(component ${ImageMagick_FIND_COMPONENTS}
     )
   if(component STREQUAL "Magick++")
     FIND_IMAGEMAGICK_API(Magick++ Magick++.h
-      Magick++ CORE_RL_Magick++_
+      Magick++ CORE_RL_Magick++_ Magick++-Q16 Magick++-Q8
       )
     list(APPEND ImageMagick_REQUIRED_VARS ImageMagick_Magick++_LIBRARY)
   elseif(component STREQUAL "MagickWand")
     FIND_IMAGEMAGICK_API(MagickWand wand/MagickWand.h
-      Wand MagickWand CORE_RL_wand_
+      Wand MagickWand CORE_RL_wand_ MagickWand-Q16 MagickWand-Q8
       )
     list(APPEND ImageMagick_REQUIRED_VARS ImageMagick_MagickWand_LIBRARY)
   elseif(component STREQUAL "MagickCore")
     FIND_IMAGEMAGICK_API(MagickCore magick/MagickCore.h
-      Magick MagickCore CORE_RL_magick_
+      Magick MagickCore CORE_RL_magick_ MagickCore-Q16 MagickCore-Q8
       )
     list(APPEND ImageMagick_REQUIRED_VARS ImageMagick_MagickCore_LIBRARY)
   else()
